plugins/modules/filter.py

```diff
@@ -44,6 +44,14 @@
 '''
 
 EXAMPLES = '''
+- name: Create a filter
+  filter:
+    name: filter
+    action: deny
+    expressions:
+      - event.check.interval == 10
+      - event.check.occurrences == 1
+    runtime_assets: awesomeness
```

Comment:
It's nice to have one simple example like the one above, but it would also be great to provide at least one "real world" example from https://docs.sensu.io/sensu-go/5.13/reference/filters/ , for example this one https://docs.sensu.io/sensu-go/5.13/reference/filters/#built-in-filter-only-incidents

Reply:
But this is actually a pipe handler that uses built-in filter `is_incident`. :sweat_smile:

Reply:
I see, the 2nd link is wrong. I actually meant this one https://docs.sensu.io/sensu-go/5.13/reference/filters/#handling-repeated-events

Reply:
I added this one and few other examples.